cpp/oneapi/dal/graph/directed_adjacency_vector_graph.hpp

```diff
@@ -51,23 +51,16 @@ class ONEDAL_EXPORT directed_adjacency_vector_graph {
     static_assert(detail::is_valid_edge_value_v<EdgeValue>,
                   "Use empty_value, double or int32_t for edge value type");
 
-    /// Constructs an empty directed_adjacency_vector_graph
+    /// Constructs an empty graph
     directed_adjacency_vector_graph();
 
+    /// Destructs the graph
     ~directed_adjacency_vector_graph() = default;
 
-    /// Move constructor for directed_adjacency_vector_graph
+    /// Move the constructor for the graph
```

Comment:
is this similar to the "move operator", in other words, should this be "the move constructor" ?